docs/use-cases/Intel-Discrete-GPU-passthrough-and-Kata.md

```diff
@@ -0,0 +1,274 @@
+In this mode of operation, the GPU is accessed exclusively by the Intel driver running in
+the VM to which it is assigned. The GPU is not shared among VMs.
+
+With SRIOV mode, it is possible to pass a Virtual GPU instance to a virtual machine.
+With this, multiple Virtual GPU instances can be carved out of a single physical GPU 
+and be passed to different VMs, allowing the GPU to be shared.
+
+| Technology | Description |
+|-|-|
+| GPU passthrough | Physical GPU assigned to a single VM |
+| SR-IOV passthrough | Physical GPU shared by multiple VMs |
+
+## Hardware Requirements
+
+Intel GPUs Recommended for Virtualization:
+
+- Intel® Data Center GPU Max Series (Ponte Vecchio)
+- Intel® Data Center GPU Flex Series (Arctic Sound-M)
+- Intel® Data Center GPU Arc Series 
+
+The following steps outline the workflow for using an Intel Graphics device with Kata.
+
+## Host BIOS requirements
+
+Hardware such as Intel Max and Flex series require larger PCI BARs. 
+
+For large BAR devices, MMIO mapping above 4G address space should be enabled in the PCI configuration of the BIOS.
+
+Some hardware vendors use a different name in BIOS, such as:
+
+- Above 4G Decoding
+- Memory Hole for PCI MMIO
+- Memory Mapped I/O above 4GB
+
+## Host Kernel Requirements
+
+For device passthrough to work with the Max and Flex Series, an out of tree kernel driver is required.
+
+For Ubuntu 22.04 server, follow these instructions to install the out of tree GPU driver:
+```bash
+$ sudo apt update
+$ sudo apt install -y gpg-agent wget
+$ wget -qO - https://repositories.intel.com/gpu/intel-graphics.key | \
+    sudo gpg --dearmor --output /usr/share/keyrings/intel-graphics.gpg
+$ echo "deb [arch=amd64 signed-by=/usr/share/keyrings/intel-graphics.gpg] https://repositories.intel.com/gpu/ubuntu ${VERSION_CODENAME}/lts/2350 unified" | \
+    sudo tee /etc/apt/sources.list.d/intel-gpu-${VERSION_CODENAME}.list
+$ sudo apt update
+$ sudo apt install -y linux-headers-$(uname -r) flex bison intel-fw-gpu intel-i915-dkms xpu-smi
+$ sudo reboot
+```
+For support on other distributions, please refer to [DGPU-docs](https://dgpu-docs.intel.com/driver/installation.html)
+
+You can also install the driver from source which is maintained at [intel-gpu-i915-backports](https://github.com/intel-gpu/intel-gpu-i915-backports)
+Detailed instructions for reference can be found at: https://github.com/intel-gpu/intel-gpu-i915-backports/blob/backport/main/docs/README_ubuntu.md
+
+Below are the steps for installing the driver from source:
+```bash
+$ export I915_BRANCH="backport/main"
+$ git clone -b ${I915_BRANCH} --depth 1 https://github.com/intel-gpu/intel-gpu-i915-backports.git && cd intel-gpu-i915-backports/
+$ sudo apt install dkms make debhelper devscripts build-essential flex bison mawk
+$ sudo apt install linux-headers-"$(UBUNTU_22.04_SERVER_KERNEL_VERSION) linux-image-unsigned-"$(UBUNTU_22.04_SERVER_KERNEL_VERSION)"
```

Comment:
These variable names are not valid. I understand why we're doing this, but I still think it would be clearer to:

- State at the top of the doc that we're assuming you are running on a Ubuntu 22.04 LTS system.
- Change this code to reference the output of `uname -r`.
- State here in a note that if you are not running on that version of Ubuntu, you will need to manually determine the latest 22.04 kernel version.